checkbox-provider-ce-oem/tests/test_rpmsg_tests.py

```diff
@@ -70,3 +73,48 @@ def test_arm_processor_unexpected(self):
                 cpu_family = rpmsg_tests.detect_arm_processor_type()
 
         self.assertEqual("unknown", cpu_family)
+
+    def test_rpmsg_test_supported_with_imx(self):
+        """
+        RPMSG TTY test is supported for i.MX
+        """
+        pattern, cmd = rpmsg_tests.rpmsg_tty_test_supported("imx")
+        self.assertEqual(pattern, r"ttyRPMSG[0-9]*")
+        self.assertEqual(cmd, "modprobe imx_rpmsg_tty")
+
+    def test_rpmsg_test_supported_with_ti(self):
+        """
+        RPMSG TTY test is not supported for TI
+        """
+        with self.assertRaisesRegex(
+                SystemExit, "Unsupported method for TI."):
+            rpmsg_tests.rpmsg_tty_test_supported("ti")
+
+    def test_rpmsg_test_supported_with_other(self):
+        """
+        RPMSG TTY test is not supported for Other SoC
+        """
+        with self.assertRaisesRegex(
+                SystemExit, "Unexpected CPU type."):
+            rpmsg_tests.rpmsg_tty_test_supported("mtk")
+
+    @patch("rpmsg_tests.check_rpmsg_tty_devices")
+    def test_no_rpmsg_devices(self, mock_check_rpmsg_tty_devices):
+        """
+        No RPMSG TTY devices not found and raise SystemExit
```

Comment:
`No RPMSG TTY devices found and raise SystemExit` or ` RPMSG TTY devices not found and raise SystemExit`, right? I prefer the last one.

Reply:
fixed